autoware_api_launch/launch/include/internal_api_relay.launch.xml

```diff
@@ -1,9 +1,9 @@
 <launch>
   <group>
     <push-ros-namespace namespace="internal"/>
-    <node pkg="topic_tools" exec="relay" name="traffic_light_states">
-      <param name="input_topic" value="/api/autoware/set/traffic_light_states"/>
-      <param name="output_topic" value="/external/traffic_light_recognition/traffic_light_states"/>
+    <node pkg="topic_tools" exec="relay" name="traffic_signals">
+      <param name="input_topic" value="/api/autoware/set/traffic_signals"/>
```

Comment:
@wep21 Is it okay to change it?

Reply:
@taichiH @wep21 This is an internal API, so it's OK if you reflect in internal tools.